docs/Reference.md

```diff
@@ -318,6 +318,7 @@ Discord uses ids and hashes to render images in the client. These hashes can be
 | Sticker Pack Banner    | app-assets/710982414301790216/store/[sticker_pack_banner_asset_id](#DOCS_RESOURCES_STICKER/sticker-pack-object).png                                                                                                                                               | PNG, JPEG, WebP      |
 | Team Icon              | team-icons/[team_id](#DOCS_TOPICS_TEAMS/team-object)/[team_icon](#DOCS_TOPICS_TEAMS/team-object).png                                                                                                                                                              | PNG, JPEG, WebP      |
 | Sticker                | stickers/[sticker_id](#DOCS_RESOURCES_STICKER/sticker-object).png \*\*\* \*\*\*\*                                                                                                                                                                                 | PNG, Lottie          |
+| Role Icon              | icons/[guild_id](#DOCS_RESOURCES_GUILD/guild-object)/roles/[role_icon](#DOCS_TOPICS_PERMISSIONS/role-object).png \*                                                                                                                                               | PNG, JPEG            |
```

Comment:
I can not confirm if this is right.
Can't set role icons yet ;)

Reply:
@advaith1 I bet you know something

Reply:
its /role-icons/role_id/role_icon.png

Reply:
ok thanks Jup ❤️